XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Pointers/LinePointer.cs

```diff
@@ -6,13 +6,21 @@
 using XRTK.Utilities.Lines.DataProviders;
 using XRTK.Utilities.Lines.Renderers;
 
+#if PLATFORM_LUMIN
+using UnityEngine.XR.MagicLeap;
```

Comment:
As with comment, can we remove the Magic leap haptics elements and we'll review for a separate framework PR for haptics

Reply:
Agree, this was kind of a fast-hack to get it up and running for our project, and not at all ideal. Stephen had started on a haptics implementation a few months ago, but it was never finished/implemented.

I can take a quick pass at something, and/or take a look at what Stephen started.